webpay/auth/decorators.py

```diff
@@ -132,11 +132,6 @@ def get_locked_step(request, step):
     :rtype: HttpResponse or None or False
 
     """
-    try:
-        step_index = flow['locked'].index(step)
-    except ValueError:
-        step_index = -1
-
```

Comment:
wfm

> On Apr 9, 2015, at 1:09 PM, Kumar McMillan <notifications@github.com> wrote:
> 
> In webpay/auth/decorators.py <https://github.com/mozilla/webpay/pull/627#discussion_r28097779>:
> 
> > @@ -132,11 +132,6 @@ def get_locked_step(request, step):
> >      :rtype: HttpResponse or None or False
> >  
> >      """
> > -    try:
> > -        step_index = flow['locked'].index(step)
> > -    except ValueError:
> > -        step_index = -1
> > -
> All this will hopefully get a proper cleanup in https://bugzilla.mozilla.org/show_bug.cgi?id=1130465 <https://bugzilla.mozilla.org/show_bug.cgi?id=1130465>
> —
> Reply to this email directly or view it on GitHub <https://github.com/mozilla/webpay/pull/627/files#r28097779>.
>